opentok/opentok-meet
# [OPENTOK-44701] Adding status callback url for WVC

#### What is this PR doing?
Adding status callback endpoint  for webview composer.

#### How should this be manually tested?
- Update config.json to use meet.tokbox.com apiKey.
- Run the app locally: `npm start`
- Use ngrok to provide you a https endpoint: `ngrok http 3000`
- Join a session and click on Webcomposer.
- Check a new textbox appears with string:
_Starting Webview Composer..._ 
and then
 _Webview composing active. Publishing here_
- Check after around 5 secs the text changes to: _Webview composing active with streamId: {streamId}. Publishing here_ 

#### What are the relevant tickets?
OPENTOK-44701
